deploy.sh

```diff
@@ -12,23 +17,27 @@ cd package
     pip install -r ../requirements.txt --target .
 cd ..
 
-# Package up lambda code
+# Package up lambda code example:
+# --s3-bucket dev2-useast1-lambda-deploy
+# --profile dev2 --region us-east-1
+
 echo "Packaging up lambda for deployment..."
 aws cloudformation package \
     --template-file template.yaml \
-    --s3-bucket dev2-useast1-lambda-deploy \
+    --s3-bucket $deploy_bucket \
     --s3-prefix lambda-healthcheck-editor \
     --output-template-file packaged-template.yaml \
-    --profile dev2
+    --profile $profile --region $region
 RETCODE=$?
 if [ $RETCODE -ne 0 ]; then
     echo "Failed to create package"
     exit $RETCODE
 fi
 
 # Deploy the lambda
+# --parameter-overrides Region=us-east-1 DomainName=https://example.com
 echo "Deploying lambda..."
-aws cloudformation deploy     --capabilities CAPABILITY_IAM  --template-file ./packaged-template.yaml  --stack-name route53-disable-healthcheck-on-failover    --parameter-overrides Region=us-east-1 --profile dev2  --region us-east-1
+aws cloudformation deploy --capabilities CAPABILITY_IAM  --template-file ./packaged-template.yaml  --stack-name route53-disable-healthcheck-on-failover  --parameter-overrides $parameters --profile $profile  --region $region
```

Comment:
Should be make the stack name an input parameter and hence configurable via this deploy script?